### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Create Web Site-Add Bindings_Screen Reader: Narrator/NVDA is not reading the labels of the input elements present on the page.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#136]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA

**MAS ID Violated**: MAS1.3.1

**Repro Steps:** 
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "Create Web Site" expandable button and Press Enter.
6. Navigate to "bindings" page tab.
7. Tab till "Add" button and Press Enter.
8. Verify that screen reader is reading the label of all the input form elements of the page.

**Actual Result:**
Narrator/NVDA is not reading the label of the following input form elements:
"Host Name", "IP Address", "Port" input fields, HTTP and Custom Control toggle buttons, Protocol and Binding Information input fields, require SNI Checkbox.

**Expected Result:**
Screen Reader should read the label of all the input form elements of the page.

**User Impact:**
Screen reader users will be unable to know the labels of the form field and hence they will not be able to provide inputs to the form fields.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment:**
[Bug 136_screen reader is not reading the labels of the input fields.pptx](https://github.com/Microsoft/IIS.WebManager/files/2327574/Bug.136_screen.reader.is.not.reading.the.labels.of.the.input.fields.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
